### ISSUE TITLE
Use standardized jshint/eslint configs

### ISSUE THREAD

**Issue by jeremyhansonfinger** [Shopify/rorybot#12]

**Comment by jeremyhansonfinger** [comment]
@nwtn would you say your changes resolved this?

**Comment by nwtn** [comment]
nope. eslint is running, but afaik it’s not using shopify’s ruleset yet. https://github.com/Shopify/javascript

**Comment by jeremyhansonfinger** [comment]
@nwtn  Ah, gotcha. Okay, I'll add that to the list.

**Comment by jeremyhansonfinger** [comment]
Oh man, ✖ 88942 problems (7290 errors, 81652 warnings)

Not surprising, I guess, since we forked an existing repo.
